modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,287 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
Do we want to outline in this documentation why you would want to use an extra bridge? Some background:

OVN-Kubernetes traditionally puts the default gateway interface (default node ingress/egress interface) onto an OVS bridge called br-ex. This interface and bridge are used as the default traffic path for pod egress traffic. This allows OVN-Kubernetes to operate in Shared gateway mode, where traffic can be directly forwarded out of OVS and bypass the kernel networking stack. Note, in Local gateway mode the bridge br-ex is also present, but typically traffic is forwarded into the kernel networking stack and then routed out of the br-ex bridge.

The Multiple External Gateways (MEG) feature allows a user to declare gateways outside of the OCP node to be used by selected pods as their next hops (rather than using the default gateway route). This allows a pod to have multiple gateways, and provides ECMP routing with BFD support. Configuration of MEG is done using a CRD known as AdminPolicyBasedExternalRoute: https://docs.redhat.com/en/documentation/openshift_container_platform/4.9/html/networking/multiple-networks#additional-network-considerations

When MEG is enabled, traffic is forwarded directly via OVS br-ex and bypasses the kernel as a forwarding next hop. This is true regardless of gateway mode.

There is a potential use case where a user may want to have egress traffic for pods affected by MEG to egress a different interface on the OCP node. In this case a user can configure a second OVS bridge called "br-ex1". This bridge will only be used for MEG egress traffic when configured. Other egress traffic for pods not configured for MEG will still by default be routed out br-ex.